client-src/index.js

```diff
@@ -115,12 +115,23 @@ self.addEventListener("beforeunload", () => {
   status.isUnloading = true;
 });
 
-const trustedTypesPolicyName =
-  typeof options.overlay === "object" && options.overlay.trustedTypesPolicyName;
-
-const overlay = createOverlay({
-  trustedTypesPolicyName,
-});
+const overlay = options.overlay
+  ? createOverlay(
+      typeof options.overlay === "object"
+        ? {
+            trustedTypesPolicyName: options.overlay.trustedTypesPolicyName,
+            catchRuntimeError: options.overlay.errors,
```

Comment:
Also ideally we need a new option for such stuff, i.e. `catchRuntimeError`, because as you can see some application works in very differents envs and it can be errors outside own application